srcpkgs/libgaminggear/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'libgaminggear'
+pkgname=libgaminggear
+version=0.15.1
+revision=1
+build_style=cmake
+configure_args="-DINSTALL_CMAKE_MODULESDIR=/usr/share/cmake
+	-DINSTALL_PKGCONFIGDIR=/usr/share/pkgconfig"
+hostmakedepends="pkg-config glib-devel"
+makedepends="glib-devel gtk+-devel libnotify-devel sqlite-devel
```

Comment:
libglib-devel should be used in makedepends= instead of glib-devel, glib-devel holds tools and utilities which can only be used in hostmakedepends=